search-app/src/cmr/search/services/humanizers/humanizer_report_service.clj

```diff
@@ -56,13 +57,25 @@
   [context rfms]
   (map #(rfm->umm-collection context %) rfms))
 
+(defn- refresh-and-get-collections-from-cache
+   "Force the collection cache to be populated and then get all the collections from it."
+   [context]
+   (let [_ (metadata-cache/refresh-collections-metadata-cache-job)
```

Comment:
We can just put `(metadata-cache/refresh-collections-metadata-cache-job)` above the `let` block, yes?